staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/options.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package schema
+
+import (
+	"strconv"
+	"strings"
+)
+
+// SchemaOptions allow for tracking paths to unknown fields.
+type SchemaOptions struct {
+	Strict            bool
```

Comment:
it's not very clear what "Strict" means... ReturnPruned was clearer, in my opinion